app/agents/voice/automatic/processors/llm_spy.py

```
@@ -332,3 +351,53 @@ async def process_frame(self, frame: Frame, direction: FrameDirection):
+            if self._context_aggregator:
+                try:
+                    context = self._context_aggregator._user._context
+                    context_messages = context.get_messages()
+
+                    for msg in reversed(context_messages):
+                        if msg.get("role") == "user":
+                            content = msg.get("content", "")
+                            if isinstance(content, str) and content.strip():
+                                user_message = content.strip()
+                                break
+                except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)